i18n/en.json

```diff
@@ -47,10 +47,6 @@
     "id": "September",
     "translation": "September"
   },
-  {
-    "id": "actiance.export.marshalToXml.appError",
-    "translation": "Unable to convert export to XML."
-  },
```

Comment:
There are a lot of unrelated changes in the file

Reply:
yeah. im not sure. i ran ```make i18n-extract```